version.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
Elsewhere I've been been moving from bash to sh for build scripts, largely for compatibility with containerized build enviroments that may not have bash available.  E.g, see https://github.com/gravitational/pithos-app/blob/04101aa199cccb19e6baac752c871f7dc7ab477e/version.sh

Doesn't need to be fixed herein, just something to keep in mind if you add version scripts like this.